src/components/NavBar.js

```diff
@@ -5,10 +5,10 @@ import {AuthContext} from "../context/AuthContext";
 
 function NavBar() {
   const navigate = useNavigate();
-  const {isAuth} = useContext(AuthContext);
+  const {auth} = useContext(AuthContext);
 
   function clickHandler() {
-      isAuth===true?navigate("/profile"):navigate("/signup")
+      auth.isAuth===true?navigate("/profile"):navigate("/signup")
```

Comment:
`BOOLEAN_WAARDE === true` is altijd hetzelfde als `BOOLEAN_WAARDE`. Dus dit kun je versimpelen:
```suggestion
      auth.isAuth ? navigate("/profile") : navigate("/signup");
```